### ISSUE TITLE
Issue with other tweaks

### ISSUE THREAD

**Issue by thoughtgap** [thomasfinch/Dim#8]
I had the tweak "Airplay server" from Bigboss repo installed and Dim didn't work. After uninstall of the other tweak it
works fine. Is there any way to make them work together?

**Comment by thomasfinch** [comment]
Unfortunately I don't have an iOS 7 device to test it on, but I'll try to figure it out. What device and version are you using it on?

**Comment by thoughtgap** [comment]
Hi, I am not so sure anymore if it really has to do something with the other tweak. I've just noticed that Dim turned on after I uninstalled the other and respringed and then came to this conclusion. Now, I am not able to turn Dim off again.
The "Activate" switch does nothing, neither does changing the brightness bar do anything... I am on an iPhone5 with iOS 7.1.2. I don't have many tweaks installed, don't even have Activator. Am I supposed to respring or restart the device in order to apply the changes? I'm happy to help if you need anything for analysis.

**Comment by thomasfinch** [comment]
No, you don't have to respring for anything in dim to take effect. That definitely sounds like a bug in Dim, I'll look into it.
